### DISCUSSION TITLE
Using x:Name works on UWP but not with WASM

### DISCUSSION THREAD

**Discussion by saadsharif** [unoplatform/uno#13997] [Q&A] [answered]
For an app that works on UWP head, I cannot get rid of this error for on the WASM head of a solution, 'The name __ does not exist in the current context'. This is to reference a SkiaSharp control.

I have tried checking the packages, checking documentation, samples, etc. 

Is it because of the use of UWP XAML and not WinUI? I cannot find a sample that makes use of UWP XAML and not WinUI.

**Comment by Youssef1313** [comment]
Can you please provide a repro project where this happens?

**Reply by saadsharif** [reply]
This is a small sample where you can see the issue. The issue is with the SkiaSharp control which is being referenced as CanvasView which causes an error.  

https://github.com/saadsharif/UnoSample

**Reply by jeromelaban** [reply, marked as answer]
Thanks for the report. This is likely caused by the fact that the namespace you added is called "skia" (Thanks @Youssef1313 
 for https://github.com/unoplatform/uno/issues/14004), which conflicts with the [platform-specific XAML feature](https://platform.uno/docs/articles/platform-specific-xaml.html).

Try using a different name, such as `sk`.
